dtm0/service.c

```diff
@@ -325,6 +377,222 @@ m0_dtm0_service_find(const struct m0_reqh *reqh)
 	return rh_srv == NULL ? NULL : to_dtm(rh_srv);
 }
 
+/* --------------------------------- EVENTS --------------------------------- */
+
+struct m0_semaphore g_test_wait;
```

Comment:
This was introduced by me just to cover unittests.
The code around this semaphore has to be revised.

Reply:
Added "dtm0 in UT" condition.